dev/io.openliberty.jakarta.validation.v31_fat/test-applications/val31/src/val31/web/Validation31TestServlet.java

```diff
@@ -52,14 +55,23 @@ public void basicRecordTest() throws Exception {
     public void recordValidatePropertyAndValueTest() throws Exception {
         // TODO create a test using the Validator.validateProperty and Validator.validateValue methods
         // on a record
+        Person propertydata = new Person(null);
+        Set<ConstraintViolation<Person>> propertyViolations = validator.validateProperty(propertydata, "name");
+        Set<ConstraintViolation<Person>> valueViolations = validator.validateValue(Person.class, "name", null);
+        assertTrue("Record Person(null) should have validated with one violation", propertyViolations.size() == 1);
+        assertEquals(1, valueViolations.size());
     }
 
     /**
      * Test that the result of a method on a record can be validated.
      */
+
     @Test
     public void recordMethodValidationTest() throws Exception {
         // TODO Add a method that includes validation to a record class and confirm that it can be validated
+        String name = "x";
+        Person propertydata = new Person(name);
+        assertFalse(propertydata.checkLength());
```

Comment:
This is just testing the method, not using validation. This one is harder than I thought, so I'll discuss it with you outside of this review.

Reply:
Updated- as per our slack comments